src/index.html

```diff
@@ -1,13 +1,309 @@
+            <li>
+              <a class="top-bar__item" href="#">Blog</a>
+            </li>
+            <li>
+              <a class="top-bar__item top-bar__item--contacts phone" href="tel:(629) 555-0129">
+                <img
+                class="phone__icon"
+                src="/images/icons/Icon-phone.png"
+                alt="Contact us"
+                >
+                Contact us
+              </a>
+            </li>
+          </ul>
+        </div>
+        <a href="#menu" class="top-bar__icon icon--menu"></a>
+      </div>
+      <!-- #endregion -->
+
+      <!-- #region header -->
+      <div class="container">
+        <div class="header__content">
+          <p class="header__motto">We bake your fantasies — croissants, cakes, cookies, and cupcakes.</p>
+          <h1 class="header__title">Creative bakery</h1>
+          <img
+            class="header__image"
+            src="/images/Cookie.png"
+            alt="Creative bakery"
+          >
+          <a href="#about" class="button header__button">Learn more</a>
+        </div>
+      </div>
+      <!-- #endregion -->
+    </header>
+    <!-- #region menu -->
+    <aside class="menu page__menu" id="menu">
+      <div class="menu__top top-bar">
+        <a href="#" class="top-bar__logo">
+          <img
+          class="menu__logo"
+          src="/images/icons/Logo.png"
+          alt="Creative Bakery"
+          >
+        </a>
+        <a href="#" class="icon icon--close"></a>
+      </div>
+      <div class="container">
+        <div class="menu__bottom">
+          <nav class="menu__nav nav">
+            <ul class="nav__list">
+              <li>
+                <a class="nav__item" href="#products">Products</a>
+              </li>
+              <li>
+                <a class="nav__item" href="#about">About Us</a>
+              </li>
+              <li>
+                <a class="nav__item" href="#">Blog</a>
+              </li>
+              <li>
+                <a class="nav__item nav__item--contacts phone" href="tel:(629) 555-0129">
+                  <img
+                  class="phone__icon"
+                  src="/images/icons/Icon-phone.png"
+                  alt="Contact us"
+                  >
+                  Contact us
+                </a>
+              </li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+    </aside>
+    <!-- #endregion -->
+    <!-- #region main -->
+    <main class="main">
+      <div class="container">
+        <div class="main__content">
+          <!-- #region products -->
+          <section class="products" id="products">
+            <h2 class="products__name">What we bake</h2>
+            <p class="products__add">We can customize our cookies right for your needs. Just imagine it</p>
+            <div class="products__card card card--1">
+              <img
+              class="card__img"
+              src="/images/photo-product-01.png"
+              alt="Cakes"
+              >
```

Comment:
Need to fix it everywhere
```suggestion
              <img
                class="card__img"
                src="/images/photo-product-01.png"
                alt="Cakes"
              >
```